examples-esp32c3/Cargo.toml

```diff
@@ -7,19 +7,29 @@ license = "MIT OR Apache-2.0"
 
 [profile.release]
 debug = true
+lto = false
+
+[profile.release.package.esp-wifi]
+opt-level = 3
+
+[profile.dev]
+lto = false
+
+[profile.dev.package.esp-wifi]
+opt-level = 3
 
 [dependencies]
-hal = { package = "esp32c3-hal", version = "0.8.0" }
+hal = { package = "esp32c3-hal", version = "0.10.0" }
 esp-backtrace = { version = "0.6.0", features = ["esp32c3", "panic-handler", "print-uart", "exception-handler"] }
 esp-println       = { version = "0.4.0", features = ["esp32c3","log"] }
 
 embassy-time = { version = "0.1.0", features = ["nightly"], optional = true }
-embassy-executor  = { package = "embassy-executor", git = "https://github.com/embassy-rs/embassy/", rev = "cd9a65b", features = ["nightly", "integrated-timers"], optional = true }
-embassy-net = { git = "https://github.com/embassy-rs/embassy", rev = "26474ce6eb759e5add1c137f3417845e0797df3a", features = ["nightly", "tcp", "udp", "dhcpv4", "medium-ethernet"], optional = true }
+embassy-executor = { version = "0.2.0", package = "embassy-executor", features = ["nightly", "executor-thread", "integrated-timers", "arch-xtensa"], optional = true }
```

Comment:
Probably copy+paste: We need `"arch-riscv32"` instead of `"arch-xtensa"`

Reply:
Yeah, it slipped in with a copy + paste. Thanks for pointing it out.